internal/controllers/inventory_controller.go

```diff
@@ -321,50 +323,69 @@ func (t *reconcilerTask) setupDeploymentManagerServerConfig(ctx context.Context,
 	return
 }
 
-// setupAlarmSubscriptionServerConfig creates the resources necessary to start the Alarm Subscription Server.
-func (t *reconcilerTask) setupAlarmSubscriptionServerConfig(ctx context.Context, defaultResult ctrl.Result) (nextReconcile ctrl.Result, err error) {
+// setupAlarmServerConfig creates the resources necessary to start the Alarm Server.
+func (t *reconcilerTask) setupAlarmServerConfig(ctx context.Context, defaultResult ctrl.Result) (nextReconcile ctrl.Result, err error) {
 	nextReconcile = defaultResult
 
-	// Create the needed ServiceAccount.
-	err = t.createServiceAccount(ctx, utils.InventoryAlarmSubscriptionServerName)
+	err = t.createServiceAccount(ctx, utils.InventoryAlarmServerName)
+	if err != nil {
+		t.logger.ErrorContext(
+			ctx,
+			"Failed to deploy ServiceAccount for the alarm server.",
+			slog.String("error", err.Error()),
+		)
+		return
+	}
+
+	err = t.createAlarmServerClusterRole(ctx)
+	if err != nil {
+		t.logger.ErrorContext(
+			ctx,
+			"Failed to create alarm server cluster role",
+			slog.String("error", err.Error()),
+		)
+		return
+	}
+
+	err = t.createAlarmServerClusterRoleBinding(ctx)
 	if err != nil {
 		t.logger.ErrorContext(
 			ctx,
-			"Failed to deploy ServiceAccount for Alarm Subscription server.",
+			"Failed to create alarm server cluster role binding",
 			slog.String("error", err.Error()),
 		)
 		return
 	}
 
 	// Create the role binding needed to allow the kube-rbac-proxy to interact with the API server to validate incoming
 	// API requests from clients.
-	err = t.createServerRbacClusterRoleBinding(ctx, utils.InventoryAlarmSubscriptionServerName)
+	err = t.createServerRbacClusterRoleBinding(ctx, utils.InventoryAlarmServerName)
 	if err != nil {
 		t.logger.ErrorContext(
 			ctx,
-			"Failed to create alarm subscription RBAC proxy cluster role binding",
+			"Failed to create alarm server RBAC proxy cluster role binding",
 			slog.String("error", err.Error()),
 		)
 		return
 	}
 
-	// Create the Service needed for the alarm subscription server.
-	err = t.createService(ctx, utils.InventoryAlarmSubscriptionServerName, utils.DefaultServicePort, utils.DefaultTargetPort)
+	// Create the Service needed for the alarm server.
+	err = t.createService(ctx, utils.InventoryAlarmServerName, utils.DefaultServicePort, utils.DefaultTargetPort)
```

Comment:
our servers are listening on port 8080. DefaultServicePort is 8000.

Reply:
i'll change them to 8000 to align with the existing servers that we already deploy to avoid further confusion.